pkgs/top-level/all-packages.nix

```diff
@@ -24740,13 +24740,21 @@ with pkgs;
 
   prospector = callPackage ../development/tools/prospector { };
 
-  protobuf = protobuf3_24;
-
-  protobuf3_24 = callPackage ../development/libraries/protobuf/3.24.nix { };
-  protobuf3_23 = callPackage ../development/libraries/protobuf/3.23.nix { };
-  protobuf3_21 = callPackage ../development/libraries/protobuf/3.21.nix {
+  protobuf = protobuf_24;
+
+  # C++ 4.24 runtime, Python 4.24 runtime
```

Comment:
I'm not sure how to interpret these comments. If this is important info it should probably be expanded upon and moved to the package definition, otherwise it will get lost with the upcoming migration to `by-name`.

Reply:
Actually, I'm not entirely satisfied with these comments. It's okay with or without them.

Reply:
Then let's drop them.